eclipse/n4js
# Gh 307

# Task
Link: https://github.com/eclipse/n4js/issues/307

In order to test this do the following:
- Open Eclipse Installer. 
- Use the Oomph setup file: https://raw.githubusercontent.com/eclipse/n4js/GH-307/releng/org.eclipse.n4js.targetplatform/N4JS.setup
- At the end of the Oomph setup process, you will eventually see the following log message in the console of the newly installed IDE:
```
0    [main] INFO  eConfigFileForSettingDefaultProduct  - File/Users/quang.tran/oomph/TestMyLaunch/ws/.metadata/.plugins/org.eclipse.pde.core/.install_folders/1508744923642/configuration/config.ini has been created successfully!
From now on, the default product to run is set to org.eclipse.n4js.product.product
```

Try to execute any Plugin UI or SWTBot test and check that `org.n4js.eclipse.product.product` is selected as product to run.
